src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/BaseStepDef.scala

```diff
@@ -213,6 +213,7 @@ class BaseStepDef extends BasePage with ScalaDsl with EN with BrowserDriver with
 
   Then("""^the user should see "(.*)" error message for "([^"]*)"$""") {
     (errorMessage: String, fieldTitle: String) =>
+      Thread.sleep(500)
```

Comment:
What is the reason for including a Thread.sleep here?

Reply:
In chrome, without this, it fails to find the error summary box. I do not know why, but I spent half a day trying to get round it with different selectors, but nothing worked. Printing the main element text did not include the error summary, so it appeared as though it was not loaded with the main html body tag.